main.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+// 各種DOMを取得
+const btn = document.querySelector('#btn');
+const textbox = document.querySelector('#textbox');
```

Comment:
ちゃんと `camelCase` にするならこんな感じです。

```suggestion
const textBox = document.querySelector('#textbox');
```